### DISCUSSION TITLE
Use ⫋  in markdown document with latex formula

### DISCUSSION THREAD

**Discussion by YievCkim** [jgm/pandoc#10118] [Q&A]
Hello,

I have a markdown document which contains some latex formula and  I can't get the ⫋ (\subsetneqq in latex) symbol. 

This is the error I got:

```
[WARNING] Missing character: There is no ⫋ (U+2ACB) (U+2ACB) in font [latinmodern-math.otf]/OT:scr
```

Is it possible to use another font ?

How can I know what is the font to use ?

I can see this font on terminal so I am sure this symbol exists somewhere...

**Comment by johnblommers** [comment]
Let me offer three ideas:

1. Look at your terminal's font configuration. My gnome terminal shows Fira Mono, and it displays ⫋  just fine.

2. The website https://www.fontspace.com/unicode/char/2ACB-subset-of-above-not-equal-to listed eight additional fonts, but not Fira Mono.

3. Install the WhatFont Firefox extension and use it to identify the name of any displayed character displayed in the web browser.

**Reply by YievCkim** [reply]
Thanks In fact I am trying to generate a pdf. And I am using Inconsolata for my terminal. I tried to add this font in header but latex didn't find it.

Anyway I managed to find a font for the rendering. This is what I have done in the markdown header:

```
mathfont: Latin Modern Math
header-includes:
 - '\setmathfont[range=\subsetneqq]{STIXTwoMath-Regular.otf}'
geometry: "margin=10mm, top=7mm"
```

It appears I have to use Stix Two Math for this specific symbol.
